templates/pages/tools/kb/knowbaseitem.html.twig

```diff
@@ -68,6 +68,24 @@
         {{ fields.nullField() }}
     {% endif %}
 
+    {{ fields.dropdownYesNo('is_token_url',
+        item.fields['is_token_url'],
+        __('Share with a unique URL'),
+        {
+            'full_width': true
+        }) }}
+    {% if item.fields['is_token_url'] %}
+        {% set token_url = config('url_base') ~ '/front/helpdesk.faq.php?id=' ~ item.fields['id'] ~ '&token=' ~ item.fields['token'] %}
+        {{ fields.htmlField('', token_url, __('Unique URL')) }}
```

Comment:
![image](https://github.com/user-attachments/assets/4e5fb2c4-6c38-4265-adec-2c61204ef88e)

Maybe you could put the url into a readonly input with a copy action like we do here ?

![image](https://github.com/user-attachments/assets/2c246d8b-0649-4a5e-9f23-e93fb907422c)